client/jetpack-cloud/sections/pricing/index.node.ts

```diff
@@ -0,0 +1,13 @@
+import { getLanguageRouteParam } from '@automattic/i18n-utils';
+import { makeLayout, setLocaleMiddleware } from 'calypso/controller';
+import { serverRouter } from 'calypso/server/isomorphic-routing';
+
+export default ( router: ReturnType< typeof serverRouter > ) => {
+	const lang = getLanguageRouteParam();
+
+	router(
+		[ `/${ lang }/pricing(/*)?`, `/${ lang }/plans(/*)?` ],
+		setLocaleMiddleware(),
```

Comment:
It may be worth adding a comment explaining what we're doing here. It's not very common to SSR a route solely for the purpose of early loading of the translations at server render.